eng/pipelines/pr.yml

```diff
@@ -116,16 +116,72 @@ stages:
           agentPool: VSEngSS-MicroBuild2022-1ES
         testType: Functional
         testTargetFramework: net472
-        timeoutInMinutes: 60
+        testProjectName: Msbuild.Integration.Test
 
 - ${{ if eq(parameters.RunFunctionalTestsOnWindows, true) }}:
   - stage:
-    displayName: Functional Tests on Windows (.NET 8.0)
+    displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.FuncTest
     dependsOn: []
     jobs:
     - template: pr.job.yml
       parameters:
-        displayName: Functional Tests on Windows (.NET 8.0)
+        displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.FuncTest
+        osName: Windows
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          agentPool: NetCore-Public
+          agentDemands: ImageOverride -equals Windows.VS2022Preview.Amd64.Open
+        ${{ else }}:
+          agentPool: VSEngSS-MicroBuild2022-1ES
+        testType: Functional
+        testTargetFramework: net472
+        testProjectName: NuGet.CommandLine.FuncTest
+
+- ${{ if eq(parameters.RunFunctionalTestsOnWindows, true) }}:
+  - stage:
+    displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.Test
+    dependsOn: []
+    jobs:
+    - template: pr.job.yml
+      parameters:
+        displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.Test
+        osName: Windows
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          agentPool: NetCore-Public
+          agentDemands: ImageOverride -equals Windows.VS2022Preview.Amd64.Open
+        ${{ else }}:
+          agentPool: VSEngSS-MicroBuild2022-1ES
+        testType: Functional
+        testTargetFramework: net472
+        testProjectName: NuGet.CommandLine.Test
+        testVerbosity: normal
+        timeoutInMinutes: 45
```

Comment:
There's been 30 and 60 minute timeouts. Is the thinking here that 60 split 2 ways is 30 plus cushion?
Or is it a measured value?

Reply:
After splitting this individual test assembly out, I ran it a few times and it always finished in 30-40 minutes.  My aim is to make the timeouts just above average so that it times out sooner rather than later.